OBJC_INTEROP.md

```diff
@@ -67,6 +67,10 @@ as a Swift extension, because Kotlin has no concept of extension constructors.
 
 Kotlin constructors are imported as initializers to Swift/Objective-C. 
 
+### Setters
+
+Reassignable Objective-C properties, that are overrides of the read-only properties, are being represented in Kotlin/Native as read-only properties, with public `set()` function.
```

Comment:
Maybe smth like

Writeable Objective-C properties, overriding read-only properties of the superclass are represented as `setFoo` method for the property `foo`.

Reply:
Also please add the following (somewhere in inheritance section):

When implementing Objective-C interface with multiple methods sharing the same signature and only differing by selectors (such as `CBCentralManagerDelegateProtocol`), please use `@Suppress("CONFLICTING_OVERLOADS")` annotation on the class implementing such an interface.

Reply:
I've made some additions, please have a look. About conflicting overloads, it was already described in the "Subclassing" section.